### ISSUE TITLE
Many X16 samples do not compile with various "label not found" errors

### ISSUE THREAD

**Issue by ciplogic** [EgonOlsen71/basicv2#44]
Hello, and thank you for the work you put into BasicV2 compiler. 

This is the X16Tests.java main method and `//*` means that these lines do give errors with `master` branch. Sadly it is well ahead of my paygrade, but maybe it is just a regression so it can be reverted back:
```
		testVpoke();
		testVpeek();
		testVpokePeek();
		testColors();
		testData();
		testCharfractal();
		//* testFrog();
		//* testTi();
		testBalls();
		testCards();
		//* testDos();
		//* testMon();
		testDivide();
		testVload();
		testSave();
		testLoad();
		//* testGeos();

		testBitmapfill();
		testLine();
		testRaytracer();
		testLine640();
		testFractal256();
		testExpresso();
		testAffine();
		testVloadTest();
		testPeek();

		//* testRamPeek();
		// testBank();
		testKoala();
		testTeapots();
		testInput();
		testFastFor();
		testScreen();
		testPset();
		testXLine();
		testFrame();
		testRect();
		testChar();
		testMouse();
		//* testSignum();
		testFractalLand();
		testXtris();
		testJoy();
		testColor();
		//* testCls();
		//*testReset();
		testCircles();
		testEllipse();
		testHashMap();
```

Kind regards!

**Comment by EgonOlsen71** [comment]
Sorry for the late reply. The master branch can't really compile for the X16's latest ROMs anyway. I stopped updating the master branch to include changes to the X16-ROM a long time ago, because it was just to much work to follow the changes in the ROM all the time, especially because nobody cared that I had to rely to some things to make it all work and moved things around again and again.
Once the  X16 is "done", I'll do one final (hopefully) version.
I'll look into this issue regardless, as it should at least compile...
